### DISCUSSION TITLE
Export prometheus or influxDB

### DISCUSSION THREAD

**Discussion by whrostre** [allure-framework/allure2#2438] [Questions & Support] [answered]
Hi, 

I'm searching for a possibility to export the allure report data to prometheus or influxDB. I already saw there are export plugins in the code (PrometheusExportPlugin, InfluxDbExportPlugin), but I have no idea how to call this features over the command line. Is there a documentation for that or is it just something internally used? Over google I just found some commands which don't exist in the current version like `allure export`

Best regards,

**Comment by baev** [comment, marked as answer]
When you generate the report (in regular mode, using `allure generate`), the files `export/influxDbData.txt` (in [Line Protocol](https://docs.influxdata.com/influxdb/v2/reference/syntax/line-protocol/) format) and `export/prometheusData.txt` (in [Prometheus exposition format](https://github.com/prometheus/docs/blob/master/content/docs/instrumenting/exposition_formats.md#text-based-format)) will be generated. 

After that, you can import such files to Influx/Prometheus via API call or using CLI, e.g., the `influx write` command for Influx.

**Reply by whrostre** [reply]
Thanks, totally overlooked that!

**Reply by baev** [reply]
If you have any ideas or improvement requests related to this feature, please submit them to a separate discussion at https://github.com/orgs/allure-framework/discussions/categories/ideas-feedback. We would love to hear from you!
